mu4/domain/notation/notationtypes.h

```diff
@@ -55,6 +59,37 @@ enum class MoveSelectionType {
     Measure,
     Track
 };
+
+struct Meta {
+    QString fileName;
+    QString title;
+    QString composer;
+    QString arranger;
+    size_t partsCount = 0;
+    QPixmap thumbnail;
+};
+
+struct ScoreInfo {
```

Comment:
I think it's better to come up with a different name. Maybe `ScoreCreateOptions` or `ScoreScreateConfig`. 
that is, you need to make it clear that this structure is used to create, and not for something else, for example, to obtain information about score

Reply:
Renamed to ScoreCreateOptions